napari/_qt/qt_viewer.py

```diff
@@ -364,16 +362,8 @@ def _on_cursor(self, event):
             Event from the Qt context.
         """
         cursor = self.viewer.cursor
-        size = self.viewer.cursor_size
         if cursor == 'square':
-            if size < 10 or size > 300:
-                q_cursor = self._cursors['cross']
-            else:
-                q_cursor = QCursor(
-                    QPixmap(':/icons/cursor/cursor_square.png').scaledToHeight(
-                        size
-                    )
-                )
+            q_cursor = QCursor(square_pixmap(self.viewer.cursor_size))
```

Comment:
I guess we still can't add `square` to the `_cursors` dictionary and remove this if / else clause, because we need to be able to do that resizing, but I've never loved this solution - curious if you have ideas on how it could be improved - maybe we make a square cursor that has a "resize" method on it, and then do if square and just call resize. Might still be more complex than it is worth